File: demo/src/main/java/com/example/android/entity/Message.java
package com.example.android.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.util.Date;

@Data
@TableName("messages")
public class Message {
    @TableId(type = IdType.AUTO)
    private Integer id;
    private Long conversationId;
    private Long senderId;
    private Long receiverId;
    private String content;
    private String type;
    private String extraData;
    private Integer isRead;
    private Date createdAt;
}

File: app/src/main/java/com/example/xiaoyi/model/Message.kt
package com.example.xiaoyi.model



data class Message(
    val id: Long = 0L,//消息ID
    val conversationId: Long = 0L,//会话ID
    val senderId: Long = 0L,//发送者ID
    val receiverId: Long = 0L,//接收者ID
    val content: String = "",//消息内容
    val type: String = "",//消息类型
    val timestamp: String = "",//时间戳
    val isRead: Boolean = false,//是否已读
    val createdAt: String = ""//创建时间
)
